Scripts/Helpers.ps1

```diff
@@ -1,6 +1,6 @@
 $repositoryRoot = Join-Path $PSScriptRoot ".."
-$csProjectPath = Join-Path $repositoryRoot "Src/CSharpier/CSharpier.csproj"
-$csharpierDllPath = Join-Path $repositoryRoot "Src/CSharpier/bin/release/net5.0/dotnet-csharpier.dll"
+$csProjectPath = Join-Path $repositoryRoot "Src/CSharpier.CLI/CSharpier.CLI.csproj"
```

Comment:
Remember to update the directory name and this script if you take my suggestion re. `CSharpier.Cli`.